openedx/adg/admin_dashboard/admin_settings/models.py

```diff
@@ -1,3 +1,53 @@
 """
 Models for `admin_settings` app
 """
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from model_utils.models import TimeStampedModel
+
+from openedx.adg.lms.applications.models import BusinessLine
+
+
+class Invitation(TimeStampedModel):
+    """
+    Model to save invitations, sent out to users for becoming admin of business unit(s).
+    """
+
+    name = models.CharField(verbose_name=_('Name'), max_length=255, null=True, blank=True, )
```

Comment:
We don't need to store the name. For invitations to non-platform users, we won't have their profile information, hence we won't have their names either. Once a user is registered in the platform, the name will be fetched from the User Profile model. For the same reason, the 'name' field was not mentioned in the ticket either.